packages/tasks/src/tasks/depth-estimation/spec/output.json

```diff
@@ -0,0 +1,9 @@
+{
+	"$id": "/inference/schemas/depth-estimation/output.json",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"description": "Outputs of inference for the Depth Estimation task",
+	"type": "array",
+	"items": {
+		"description": "The output depth labels"
+	}
```

Comment:
we don't know the output format I guess?

Reply:
According to this doc: https://huggingface.co/docs/transformers/v4.37.1/en/main_classes/pipelines#transformers.DepthEstimationPipeline

The output is an image, so I specced it with the same type as image inputs

Reply:
Right, didn't know that :+1: So maybe update the description to something like `Output image representing the estimated depth.`?

It seems that the `transformers`'s dostring for [DepthEstimationPipeline](https://github.com/huggingface/transformers/blob/7fa4b36ebaebc6f5ca72ccf08ed0fe2a84422c5c/src/transformers/pipelines/depth_estimation.py#L76) is an exact copy-paste from [ImageClassificationPipeline](https://github.com/huggingface/transformers/blob/7fa4b36ebaebc6f5ca72ccf08ed0fe2a84422c5c/src/transformers/pipelines/image_classification.py#L149) (depth estimation shouldn't be returning labels _I guess_). (cc @amyeroberts maybe?)

Reply:
Iirc, the output is a dictionary with two entries, one being the `depth` which is a depth estimation image, the other is `predicted_depth`, which is the tensor. See https://huggingface.co/docs/transformers/main/tasks/monocular_depth_estimation

Reply:
https://github.com/huggingface/transformers/issues/28729